### DISCUSSION TITLE
希望能在右键菜单添加发送文件

### DISCUSSION THREAD

**Discussion by MiKoto-Railgun** [localsend/localsend#734] [L-简体中文]
最好可以适配win11右键菜单

**Comment by weiyideai520** [comment]
+1，同样需要

**Comment by yuze0804007** [comment]
+1，同样需要

**Comment by Rain-0x01-39** [comment]
+1，同样需要

**Comment by runningwaterpro** [comment]
> 最好可以适配win11右键菜单

1. Win+R，粘贴“shell:sendto”
2. 把 LocalSend 的快捷方式粘进来
3. 之后文件的右键菜单“发送到”中就有 LocalSend 了

<img width="707" height="204" alt="image" src="https://github.com/user-attachments/assets/6583624a-19a1-4fb3-b690-e9849419fd97" />

-----
<img width="424" height="87" alt="PixPin_2025-09-25_10-39-16" src="https://github.com/user-attachments/assets/230ee70c-c232-4aee-a69a-8982a5f697ca" />

**Reply by MiKoto-Railgun** [reply]
> 3. 之后文件的右键菜单“发送到”中就有 LocalSend 了

确实有了，但是打开了 LocalSend ，却没有把文件发送到页面了

**Reply by runningwaterpro** [reply]
我在 Win10、Win11 上都验证过是正常的（点击查看下面GIF演示），可能你的系统有什么特殊情况导致的吧。

还可以试试 LocalSend 自己的功能，在设置中选择“高级设置”，选中“在’发送到...‘文件菜单中显示LocalSend”选项，再试试吧。由于我的发送到是正常的，所以这个没有验证。

![PixPin_2025-09-28_17-18-31](https://github.com/user-attachments/assets/2b768154-f36b-456e-8052-4061a1e5334c)

--------

<img width="640" height="711" alt="image" src="https://github.com/user-attachments/assets/9c456c2d-5ebd-4902-8969-dac04a327a6b" />

**Reply by Rain-0x01-39** [reply]
> 我在 Win10、Win11 上都验证过是正常的（点击查看下面GIF演示），可能你的系统有什么特殊情况导致的吧。
> 
> 还可以试试 LocalSend 自己的功能，在设置中选择“高级设置”，选中“在’发送到...‘文件菜单中显示LocalSend”选项，再试试吧。由于我的发送到是正常的，所以这个没有验证。
> 
> ![PixPin_2025-09-28_17-18-31](https://github.com/user-attachments/assets/2b768154-f36b-456e-8052-4061a1e5334c)
> 
> --------
> 
> <img width="640" height="711" alt="image" src="https://github.com/user-attachments/assets/9c456c2d-5ebd-4902-8969-dac04a327a6b" />
> 

旧的右键已经有了，要是新的也有那就更好了
